sdk/core/azure-core/tests/testserver_tests/coretestserver/coretestserver/test_routes/streams.py

```diff
@@ -48,6 +54,10 @@ def string():
     return Response(
         streaming_test(), status=200, mimetype="text/plain"
     )
+
+@streams_api.route('/compress', methods=['GET'])
```

Comment:
Is this compressed stream w/o compress header?

Could you add this information into the API?